src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const firstPromise = new Promise((resolve) => {
+  document.querySelector('.logo').addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+firstPromise.then((message) => {
+  const messageDiv = document.createElement('div');
+
+  messageDiv.classList.add('message');
+  messageDiv.textContent = message;
+  document.body.appendChild(messageDiv);
+});
+
+const secondPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    reject(new Error('Promise was rejected!'));
+  }, 3000);
+});
+
+secondPromise.catch((error) => {
+  const errorDiv = document.createElement('div');
+
+  errorDiv.classList.add('message', 'error-message');
+  errorDiv.textContent = error.message;
+  document.body.appendChild(errorDiv);
+});
```

Comment:
The error handling logic is correct, but consider using 'finally' to handle any cleanup or final actions that should occur regardless of whether the promise is resolved or rejected. This is not a critical issue but could improve the robustness of your code.